packages/story-editor/src/components/hotlinkModal/useHotlinkModal.js

```diff
@@ -31,7 +34,40 @@ import {
   getErrorMessage,
 } from './utils';
 
-function useHotlinkModal({ onSelect, onClose, onError, allowedFileTypes }) {
+function CORSMessage() {
```

Comment:
Having a component in a file for a hook, now feels very strange. For tidiness sake ( not reusable ), I think this should be in it's own file. There should even be a storybook integration that way as well.